testsuite/tests/singlecluster/overrides/merge/same_target/test_ab_strategy.py

```diff
@@ -0,0 +1,27 @@
+"""Test override policies aimed at the same resoure uses oldested policy."""
+
+import pytest
+
+from .conftest import OVERRIDE_LIMIT, OVERRIDE_LIMIT2
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, route, rate_limit, override_merge_rate_limit):  # pylint: disable=unused-argument
+    """Commits RateLimitPolicy after the HTTPRoute is created"""
+    for policy in [rate_limit, override_merge_rate_limit]:  # Forcing order of creation.
+        request.addfinalizer(policy.delete)
+        policy.commit()
+        policy.wait_for_accepted()
+
+
+def test_multiple_policies_merge_default_ab(client):
+    """Test RateLimitPolicy with merge overrides being ignored due to age"""
```

Comment:
The RateLimitPolicy with merge overrides is not ignored. On the contrary it is fully enforced

Reply:
I think this behavior is correct, overrides with the merge strategy always take precedence. Some examples from the docs: https://docs.kuadrant.io/1.0.x/architecture/rfcs/0009-defaults-and-overrides/#examples-d-override-policy-rules-merged-into-other-at-lower-level